airflow/operators/latest_only_operator.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import logging
+
+from airflow.models import BaseOperator, TaskInstance
+from airflow.utils.state import State
+from airflow import settings
+
+
+class LatestOnlyOperator(BaseOperator):
+    """
+    Allows a workflow to skip tasks that are not running during the most
+    recent schedule interval.
+
+    If the task is run outside of the latest schedule interval, all
+    directly downstream tasks will be skipped.
+    """
+
+    ui_color = '#bb6bd6'  # nyanza
+
+    def execute(self, context):
+        start_date = context['ti'].start_date
```

Comment:
The reason to use `start_date` instead of `datetime.now()` is because start_date gets stored in the metadata db so we can see it in the UI